app/controllers/samples_controller.rb

```diff
@@ -223,36 +223,18 @@ def heatmap
         Viruses: ["Phage"]
       },
       metrics: [
+        "NT.r", # first entry will be default selection
```

Comment:
Better do default selection in the client. Just select the value instead of the first one, so that the metrics are ordered.

Reply:
ok